server/sandbox_run_linux.go

```diff
@@ -726,7 +726,7 @@ func AddCgroupAnnotation(ctx context.Context, g generate.Generator, mountPath, c
 				// strip off the newline character and convert it to an int
 				strMemory := strings.TrimRight(string(fileData), "\n")
 				if strMemory != "" {
-					memoryLimit, err := strconv.Atoi(strMemory)
+					memoryLimit, err := strconv.ParseInt(strMemory, 10, 64)
```

Comment:
It's pre-tea for me, but shouldn't 64 be 32 here?

https://golang.org/pkg/strconv/#ParseInt  The bitSize argument specifies the integer type that the result must fit into. Bit sizes 0, 8, 16, 32, and 64 correspond to int, int8, int16, int32, and int64.

Reply:
The point is that the integer parsed there does not fit in 32 bits.

Reply:
Right, but with it set to 64 here, does it fit?  I thought it would need to be 32 but perhaps I'm mis-reading the doc.

Reply:
Actually, my platform has:
```
# cat /sys/fs/cgroup/memory/memory.limit_in_bytes 
8796093018112
```
That does not fit in a 32 bit integer, and with that I got runtime conversion errors. Now it can parse it, and does not fail.

Reply:
My amd64 platform has even a larger value and on 64 bit systems int type has 64 bits. So there it fits.
```
$ cat /sys/fs/cgroup/memory/memory.limit_in_bytes
9223372036854771712
```